crates/fixt/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ---
-default_semver_increment_mode: !pre_patch beta-dev
+default_semver_increment_mode: !pre_minor beta-rc
```

Comment:
```suggestion
default_semver_increment_mode: !pre_patch beta-rc
```

this is suggestion applies for all other occurrences as well.

### rationale
this branch will never have any `!*minor` bump-mode by intention. that would bump to a pre-release of the next minor version i.e. `0.Y+1.Z-<suffix>.0`, in this case specifically it would bump fixt to `0.3.0-beta-rc.0.

Reply:
Done